Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/TypeStructure.cpp

```diff
@@ -372,13 +372,14 @@ TSharedPtr<FUnrealType> CreateUnrealTypeInfo(UStruct* Type, uint32 ParentChecksu
 	for (int CmdIndex = 0; CmdIndex < RepLayout.Cmds.Num(); ++CmdIndex)
 	{
 		FRepLayoutCmd& Cmd = RepLayout.Cmds[CmdIndex];
-		FRepParentCmd& Parent = RepLayout.Parents[Cmd.ParentIndex];
 
 		if (Cmd.Type == REPCMD_Return || Cmd.Property == nullptr)
 		{
 			continue;
 		}
 
+		FRepParentCmd& Parent = RepLayout.Parents[Cmd.ParentIndex];
```

Comment:
I assume this was a driveby fix?

Reply:
Yes, this was causing a crash when there were no replicated properties on a type bound class. I saw this fix already made it's way into the dynamic type bindings.